test_conformance/api/test_kernels.cpp

```diff
@@ -194,30 +194,34 @@ int test_execute_kernel_local_sizes(cl_device_id deviceID, cl_context context, c
     clKernelWrapper kernel;
     clMemWrapper            streams[2];
     size_t    threads[1], localThreads[1];
-    cl_float inputData[100];
-    cl_int outputData[100];
     RandomSeed seed( gRandomSeed );
     int i;
 
+    num_elements = 100;
+    std::vector<cl_float> inputData(num_elements);
+    std::vector<cl_int> outputData(num_elements);
+
     /* Create a kernel to test with */
     if( create_single_kernel_helper( context, &program, &kernel, 1, sample_single_test_kernel, "sample_test" ) != 0 )
     {
         return -1;
     }
 
     /* Create some I/O streams */
-    streams[0] = clCreateBuffer(context, (cl_mem_flags)(CL_MEM_READ_WRITE),  sizeof(cl_float) * 100, NULL, &error);
+    streams[0] = clCreateBuffer(context, (cl_mem_flags)(CL_MEM_READ_WRITE),
```

Comment:
We should probably remove all these `cl_mem_flags` casts at some point. Probably best done as a global sed job in a dedicated PR.

Reply:
Opened https://github.com/KhronosGroup/OpenCL-CTS/issues/759 for this.